pkg/tsdb/azuremonitor/loganalytics/azure-response-table-frame.go

```diff
@@ -112,6 +116,55 @@ var converterMap = map[string]data.FieldConverter{
 	"number":   decimalConverter,
 }
 
+type KeyValue struct {
+	Value interface{} `json:"value"`
+	Key   string      `json:"key"`
+}
+
+var tagsConverter = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableJSON,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+
+		m := map[string]interface{}{}
```

Comment:
```suggestion
		m := map[string]any{}
```
You can use `any` instead of empty interfaces